pkg/controller/secret/secret_controller_test.go

```diff
@@ -644,10 +665,11 @@ func Test_ReconcileSecrets(t *testing.T) {
 			createSecret(reconciler, secretNamePD, secretKeyPD, pdKey)
 		}
 
-		configExpected := createAlertManagerConfig(pdKey, wdURL, exampleConsoleUrl)
+		configExpected := createAlertManagerConfig(pdKey, wdURL, exampleConsoleUrl, "fake-cluster-id")
 
 		verifyInhibitRules(t, configExpected.InhibitRules)
 
+		createClusterVersion(reconciler)
```

Comment:
I think this needs to happen sooner, doesn't it? Like, line 647?

Reply:
yes, now that the test actually does something I need to move it :)